tokio/src/runtime/dump.rs

```diff
@@ -30,14 +30,187 @@ pub struct Task {
     trace: Trace,
 }
 
+/// A backtrace symbol. This is similar to [backtrace::BacktraceSymbol],
```

Comment:
Rust types should be in backticks. This way, they don't need to be in the spellcheck dictionary.
```suggestion
/// A backtrace symbol. This is similar to [`backtrace::BacktraceSymbol`],
```